src/controller/python/chip/clusters/Attribute.py

```diff
@@ -1119,25 +1134,6 @@ def Read(future: Future, eventLoop, device, devCtrl,
     return res
 
 
-def ReadAttributes(future: Future, eventLoop, device, devCtrl,
```

Comment:
There are references to these functions in:
https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/repl/Matter_Basic_Interactions.ipynb?short_path=41c1c78#L3507

Reply:
Hm, good catch! It seems to me that those comments should reference the non-plural variant on the device controller object. I've adjusted the comments:
https://github.com/project-chip/connectedhomeip/pull/35557/commits/420963a3742a461c3d3d5d95974ffd03f367edcc